app/services/proofing/mock/resolution_mock_client.rb

```diff
@@ -3,9 +3,17 @@ module Mock
     class ResolutionMockClient < Proofing::Base
       vendor_name 'ResolutionMock'
 
-      required_attributes :first_name, :ssn, :zipcode
-
-      optional_attributes :uuid, :uuid_prefix
+      required_attributes :uuid,
+                          :first_name,
+                          :last_name,
+                          :dob,
+                          :ssn,
+                          :address1,
```

Comment:
just for clarification - address1 is the street address correct?

Reply:
Yeah we populate it through the `Idv::AddressForm` (Address step):

https://github.com/18F/identity-idp/blob/17eddab5ab5a49ba83e817fede6ca8449d337c46/app/forms/idv/address_form.rb#L6

https://github.com/18F/identity-idp/blob/17eddab5ab5a49ba83e817fede6ca8449d337c46/app/forms/idv/address_form.rb#L34-L42